src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistForm.tsx

```diff
@@ -0,0 +1,297 @@
+
+        //  be sure to load the initial organization or reset it if we can't find it
+        if (formValues.organizationId) {
+          if (
+            data.results.find(
+              (element) => element.id === formValues.organizationId,
+            ) !== undefined
+          ) {
+            //  queries results contains the selected organization
+            setIsInitOrganization(true);
+          } else if (
+            currentOrganizationPage * localItemPerPage + localItemPerPage >=
+            data.count
+          ) {
+            //  we have parsed all organization and we can not find the one selected
+            setFormValues((currentValue) => ({
+              ...currentValue,
+              organizationId: undefined,
+            }));
+            setIsInitOrganization(true);
+          } else {
+            //  we have yet some page to load
+            setCurrentOrganizationPage((currentPage) => currentPage + 1);
+          }
+        } else {
+          setIsInitOrganization(true);
+        }
+      },
+    },
+  );
+  const [organizations, setOrganizations] = useState<Organization[]>([]);
+
+  useLayoutEffect(() => {
+    if (!organizationResponse) {
+      return;
+    }
+
+    setOrganizations((currentOrganizationPage) => {
+      const copie = [...currentOrganizationPage];
+      organizationResponse.results.forEach((newItem) => {
+        if (!copie.find((item) => item.id === newItem.id)) {
+          copie.push(newItem);
+        }
+      });
+
+      return copie;
+    });
+  }, [organizationResponse]);
+
+  if (!isInitOrganization && !isError) {
+    return (
+      <Box width="large">
+        <Box margin="auto">
+          <Spinner />
+        </Box>
+      </Box>
+    );
+  }
+
+  if (isError) {
+    return (
+      <Box width="large">
+        <Text>{intl.formatMessage(messages.errorOrganizations)}</Text>
+        <Box margin="auto" pad={{ top: 'medium' }}>
+          <Button
+            onClick={() => {
+              refetch();
+            }}
+            primary
+            label={intl.formatMessage(messages.retryOrganizations)}
+          />
+        </Box>
+      </Box>
+    );
+  }
+
+  return (
+    <ThemeContext.Extend value={isEditable ? {} : disableFormTheme}>
+      <Form
+        onSubmit={(event) => {
+          const values = event.value;
+
+          onSubmit(values);
+        }}
+        value={formValues}
+        onChange={(values) => {
+          setFormValues(values);
+        }}
+        onSubmitError={() => ({})}
+        messages={{
+          required: intl.formatMessage(messages.requiredField),
+        }}
+      >
+        <Box gap="medium">
+          {title && (
+            <Heading
+              size="3"
+              alignSelf="center"
+              margin={{ top: '0', bottom: 'small' }}
+            >
+              {title}
+            </Heading>
+          )}
+
+          <Box>
+            <FormField
+              required
+              disabled={!isEditable}
```

Comment:
IMO We should distinguished visually when the field is disabled.

Reply:
I could aggree, but having a grey overlay over fields when we are not editing the form is pretty horrible

Reply:
I don't understand too much the value of an Edit button so, if we cannot distinguished the difference between editable and not editable. 
Why not put the fields directly editable, and the click on edit update the fields?